### ISSUE TITLE
[Bug]: landing page reactions are not accounting user session

### ISSUE THREAD

**Issue by 0-vortex** [civitai/civitai#698]
### What happened?

Navigating to the landing page of civitai, in the "Featured Images" section noticed one of my images got promoted, but I hadn't upvoted it - after clicking the counter stayed the same, but went down if checking the actual post.

### Steps to reproduce the problem

1. Go to civitai.com landing page
2. See an image you previously liked
3. Like it again
4. The like counter goes down

### What should have happened?

The lading page featured section should correctly account for your previous reactions and provide visual feedback for it

### What platforms do you use to access the site?

Windows, MacOS, iOS

### What browsers do you use to access the site?

Google Chrome, Brave, Apple Safari

### Additional information, context and logs

Front page section:

<img width="697" alt="Screenshot 2023-09-07 at 07 59 22" src="https://github.com/civitai/civitai/assets/237133/4a83928b-d287-4faf-a6ba-8276b2047876">

Actual image post:

<img width="501" alt="Screenshot 2023-09-07 at 07 59 30" src="https://github.com/civitai/civitai/assets/237133/0ec42bc6-0b9d-41f7-a8c2-f744613f58f9">

**Comment by JustMaier** [comment]
Yup. This is a constraint with the way we're currently fetching data for those home blocks. We're aware of the problem but don't have an easy fix for it at this time without a major rework. It was an unfortunate decision at development time.

We're working on something that might help with it, but we're currently heavily focused on other areas of the product and it'll probably have to stay like this for a month or so. :(
